README.md

```diff
@@ -143,11 +156,12 @@ Here is where customers select a payment method and place the order.
 | --------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------- |
 | [Adyen](https://docs.commercelayer.io/developers/v/how-tos/payments/adyen)                    | Credit card / [PayPal](https://docs.adyen.com/payment-methods/paypal/web-drop-in) / [Klarna](https://docs.adyen.com/payment-methods/klarna/web-drop-in) | &check;         |
 | [Braintree](https://docs.commercelayer.io/developers/v/how-tos/payments/braintree)            | Credit card                                                                                                                                             | &check;         |
-| [Checkout.com](https://docs.commercelayer.io/developers/v/how-tos/payments/checkout.com)      | Credit card                                                                                                                                             | &check;         |
 | [PayPal](https://docs.commercelayer.io/developers/v/how-tos/payments/paypal)                  | PayPal                                                                                                                                                  | &cross;         |
-| [Stripe](https://docs.commercelayer.io/developers/v/how-tos/payments/stripe)                  | Credit card                                                                                                                                             | &check;         |
+| [Stripe](https://docs.commercelayer.io/developers/v/how-tos/payments/stripe)                  | Credit card / Apple Pay / Google Pay / PayPal / Klarna                                                                                                  | &check;         |
 | [Manual gateway](https://docs.commercelayer.io/developers/v/how-tos/payments/manual-payments) | Manual payment                                                                                                                                          | &cross;         |
 
+> To automatically accept payment methods enabled in the Stripe dashboard please make sure to properly [set up the auto_payments option](https://docs.commercelayer.io/core/v/api-reference/stripe_gateways) on your Stripe gateway.
```

Comment:
I would use backticks on auto_payments: `auto_payments`

Reply:
I tried, but on preview I was getting this, let's see on github how it will render.

<img width="467" alt="Screenshot 2023-09-28 at 15 15 04" src="https://github.com/commercelayer/mfe-checkout/assets/97170183/1287c026-d717-4ac8-847f-64e248b1710c">

Reply:
I see, two options then:
- simply write "auto payments"
- link the words "Stripe gateway" (which is also more correct IMO)

Reply:
I'll go for the second! thanks